docs/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.md

```diff
@@ -23,12 +23,13 @@ Onboarding of users will be done in batches. You will be notified via email when
 
 Before you can deploy your Strapi application on Strapi Cloud, you need to have the following prerequisites:
 
-* A [GitHub](https://github.com) account
-* GitHub repository for your Strapi application(s)
-* Strapi version 4.1+
+* Strapi version `4.4.x` or higher
+* Database: PostgreSQL, MySQL, or MariaDB
+* Project(s) source code hosted on [GitHub](https://github.com)
+    * The connected repository can contain multiple Strapi applications. Each Strapi app must be in a separate directory.
 
-::: tip
-The connected repository can contain multiple Strapi applications. Each Strapi app must be in a separate directory.
+::: warning
+Importing data from an existing Strapi application and data backups for a Strapi Cloud application are not supported at this time. These features are in active development and will be available in the future.
```

Comment:
Could we please remove here "_are in active development_" part, because that's not relevant for data backups right now :)
So it'd be "These features will be available in the future".
Thank you.